packages/tenant-process/test/testAddCertifiedAttribute.ts

```diff
@@ -0,0 +1,256 @@
+import { fail } from "assert";
+import { attributeKind } from "pagopa-interop-models";
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import { readLastEventByStreamId } from "pagopa-interop-commons-test/index.js";
+import {
+  generateId,
+  Tenant,
+  Attribute,
+  unsafeBrandId,
+  protobufDecoder,
+  TenantCertifiedAttributeAssignedV2,
+  fromTenantKindV2,
+  toTenantV2,
+} from "pagopa-interop-models";
+import { describe, it, expect, vi } from "vitest";
+import {
+  tenantNotFound,
+  attributeNotFound,
+  tenantIsNotACertifier,
+  certifiedAttributeOriginIsNotCompliantWithCertifier,
+  certifiedAttributeAlreadyAssigned,
+} from "../src/model/domain/errors.js";
+import { ApiCertifiedTenantAttributeSeed } from "../src/model/types.js";
+import {
+  currentDate,
+  getMockAuthData,
+  addOneAttribute,
+  addOneTenant,
+  getMockTenant,
+  getMockCertifiedTenantAttribute,
+} from "./utils.js";
+import {
+  postgresDB,
+  attributes,
+  tenants,
+  tenantService,
+} from "./tenant.integration.test.js";
+
+export const testAddCertifiedAttribute = (): ReturnType<typeof describe> =>
+  describe("addCertifiedAttribute", async () => {
+    const tenantAttributeSeed: ApiCertifiedTenantAttributeSeed = {
+      id: generateId(),
+    };
+    const correlationId = generateId();
+    const requesterTenant: Tenant = {
+      ...getMockTenant(),
+      features: [
+        {
+          type: "PersistentCertifier",
+          certifierId: generateId(),
+        },
+      ],
+      updatedAt: currentDate,
+    };
+    const attribute: Attribute = {
+      name: "an Attribute",
+      id: unsafeBrandId(tenantAttributeSeed.id),
+      kind: attributeKind.certified,
+      description: "an attribute",
+      creationTime: new Date(),
+      code: "123456",
+      origin: requesterTenant.features[0].certifierId,
+    };
+
+    const targetTenant: Tenant = getMockTenant();
+    const organizationId = getMockAuthData(requesterTenant.id).organizationId;
+
+    it("Should add the certified attribute if the tenant doesn't have that", async () => {
+      vi.useFakeTimers();
+      vi.setSystemTime(new Date());
+      await addOneAttribute(attribute, attributes);
+      await addOneTenant(targetTenant, postgresDB, tenants);
+      await addOneTenant(requesterTenant, postgresDB, tenants);
+      await tenantService.addCertifiedAttribute(targetTenant.id, {
+        tenantAttributeSeed,
+        organizationId,
+        correlationId,
+      });
+      const writtenEvent = await readLastEventByStreamId(
+        targetTenant.id,
+        "tenant",
+        postgresDB
+      );
+      if (!writtenEvent) {
+        fail("Update failed: tenant not found in event-store");
+      }
+      expect(writtenEvent).toMatchObject({
+        stream_id: targetTenant.id,
+        version: "1",
+        type: "TenantCertifiedAttributeAssigned",
+      });
```

Comment:
```suggestion
        event_version: 2,
      });
```
You forgot to check `event_version`